examples/step-3/step-3.cc

```diff
@@ -435,18 +437,19 @@ void Step3::assemble_system()
           // this is repeated for all shape functions $i$ and $j$:
           for (unsigned int i = 0; i < dofs_per_cell; ++i)
             for (unsigned int j = 0; j < dofs_per_cell; ++j)
-              cell_matrix(i, j) +=
-                (fe_values.shape_grad(i, q_index) *
-                 fe_values.shape_grad(j, q_index) * fe_values.JxW(q_index));
+              cell_matrix(i, j) += (fe_values.shape_grad(i, q_index) * //
+                                    fe_values.shape_grad(j, q_index) * //
+                                    fe_values.JxW(q_index));
```

Comment:
How about writing "grad phi_i(x_q)", "grad phi_j(x_q)" and "dx" behind the three parts? This way you actually explain something :-)